_adr/80/index.adoc

```diff
@@ -12,11 +12,80 @@ In order to provide support to customers of a Managed Service, CEE require a mec
 state of the instance of the service for diagnostic purposes.  CEE will use this information in conjunction with the
 problem statement provided by the customer in order to efficiently investigate the problem and determine a solution.
 
-The data gathering mechanism must consider the issue of Personally Identifiable Information (PII) or other sensitive
-information and ensure that it is handled appropriately.
+The data gathering mechanism must consider the issue of
```

Comment:
General comment

I think this ADR needs a different title.

Since these are open source, any obscure function name is not helping.
I propose to rename CEE with a support organization.